### ISSUE TITLE
ngDropDowns will not show list when clicked after upgrading to v1.0.0-alpha.18 

### ISSUE THREAD

**Issue by iamtom2** [ng-bootstrap/ng-bootstrap#1200]
### Bug description:
Our automated build failed its end to end tests today.  When we verified the error by manually running our application we saw that the drop down buttons on our application do longer show the associated ul lists when clicked.  After checking the log the only notable change was in the npm update:

clipping of npm update output on working build:
+-- @ng-bootstrap/ng-bootstrap@1.0.0-alpha.17 
+-- @types/node@6.0.58 

clipping of npm update output on broken build:
+-- @ng-bootstrap/ng-bootstrap@1.0.0-alpha.18 
+-- @types/node@6.0.59 

We forced our build back to the following clipping from our package.json and the bulid works fine
    "@ng-bootstrap/ng-bootstrap": "1.0.0-alpha.17",
    "@types/node": "6.0.58",

### Link to minimally-working plunker that reproduces the issue:

We haven't gone through the effort of logging this yet only because I don't know how to import my packages.json into plunker.

### Version of Angular, ng-bootstrap, and Bootstrap:
below is the broken packages.json:
{
  "name": "angular-quickstart",
  "version": "1.0.0",
  "description": "QuickStart package.json from the documentation, supplemented with testing support",
  "scripts": {
    "start": "tsc && concurrently \"tsc -w\" \"lite-server\" ",
    "docker-build": "docker build -t ng2-quickstart .",
    "docker": "npm run docker-build && docker run -it --rm -p 3000:3000 -p 3001:3001 ng2-quickstart",
    "pree2e": "npm run webdriver:update",
    "e2e": "tsc && concurrently \"protractor protractor.config.js\" --kill-others --success first",
    "lint": "tslint ./app/**/*.ts -t verbose",
    "lite": "lite-server",
    "test": "tsc && concurrently \"tsc -w\" \"karma start karma.conf.js\"",
    "test-once": "tsc && karma start karma.conf.js --single-run",
    "tsc": "tsc",
    "tsc:w": "tsc -w",
    "webdriver:update": "webdriver-manager update"
  },
  "keywords": [],
  "author": "",
  "licenses": [
    {
      "type": "MIT",
      "url": "https://github.com/angular/angular.io/blob/master/LICENSE"
    }
  ],
  "dependencies": {
    "@angular/common": "~2.1.2",
    "@angular/compiler": "~2.1.2",
    "@angular/core": "~2.1.2",
    "@angular/forms": "~2.1.2",
    "@angular/http": "~2.1.2",
    "@angular/platform-browser": "~2.1.2",
    "@angular/platform-browser-dynamic": "~2.1.2",
    "@angular/router": "~3.1.2",
    "@angular/upgrade": "~2.1.2",
    "@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.14",
    "angular-in-memory-web-api": "~0.1.13",
    "core-js": "^2.4.1",
    "reflect-metadata": "^0.1.8",
    "rxjs": "5.0.0-beta.12",
    "systemjs": "0.19.40",
    "zone.js": "^0.6.26"
  },
  "devDependencies": {
    "@types/core-js": "^0.9.34",
    "@types/jasmine": "^2.5.36",
    "@types/node": "^6.0.46",
    "@types/selenium-webdriver": "2.53.36",
    "canonical-path": "0.0.2",
    "concurrently": "^3.1.0",
    "http-server": "^0.9.0",
    "jasmine-core": "~2.4.1",
    "karma": "^1.3.0",
    "karma-chrome-launcher": "^2.0.0",
    "karma-cli": "^1.0.1",
    "karma-htmlfile-reporter": "^0.3.4",
    "karma-jasmine": "^1.0.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "lite-server": "^2.2.2",
    "lodash": "^4.16.4",
    "protractor": "4.0.9",
    "rimraf": "^2.5.4",
    "superagent": "^3.3.1",
    "tslint": "^3.15.1",
    "typescript": "^2.0.3",
    "webdriver-manager": "10.2.5"
  },
  "repository": {}
}

and the working packages.json:
{
  "name": "angular-quickstart",
  "version": "1.0.0",
  "description": "QuickStart package.json from the documentation, supplemented with testing support",
  "scripts": {
    "start": "tsc && concurrently \"tsc -w\" \"lite-server\" ",
    "docker-build": "docker build -t ng2-quickstart .",
    "docker": "npm run docker-build && docker run -it --rm -p 3000:3000 -p 3001:3001 ng2-quickstart",
    "pree2e": "npm run webdriver:update",
    "e2e": "tsc && concurrently \"protractor protractor.config.js\" --kill-others --success first",
    "lint": "tslint ./app/**/*.ts -t verbose",
    "lite": "lite-server",
    "test": "tsc && concurrently \"tsc -w\" \"karma start karma.conf.js\"",
    "test-once": "tsc && karma start karma.conf.js --single-run",
    "tsc": "tsc",
    "tsc:w": "tsc -w",
    "webdriver:update": "webdriver-manager update"
  },
  "keywords": [],
  "author": "",
  "licenses": [
    {
      "type": "MIT",
      "url": "https://github.com/angular/angular.io/blob/master/LICENSE"
    }
  ],
  "dependencies": {
    "@angular/common": "~2.1.2",
    "@angular/compiler": "~2.1.2",
    "@angular/core": "~2.1.2",
    "@angular/forms": "~2.1.2",
    "@angular/http": "~2.1.2",
    "@angular/platform-browser": "~2.1.2",
    "@angular/platform-browser-dynamic": "~2.1.2",
    "@angular/router": "~3.1.2",
    "@angular/upgrade": "~2.1.2",
    "@ng-bootstrap/ng-bootstrap": "1.0.0-alpha.17",
    "angular-in-memory-web-api": "~0.1.13",
    "core-js": "^2.4.1",
    "reflect-metadata": "^0.1.8",
    "rxjs": "5.0.0-beta.12",
    "systemjs": "0.19.40",
    "zone.js": "^0.6.26"
  },
  "devDependencies": {
    "@types/core-js": "^0.9.34",
    "@types/jasmine": "^2.5.36",
    "@types/node": "6.0.58",
    "@types/selenium-webdriver": "2.53.36",
    "canonical-path": "0.0.2",
    "concurrently": "^3.1.0",
    "http-server": "^0.9.0",
    "jasmine-core": "~2.4.1",
    "karma": "^1.3.0",
    "karma-chrome-launcher": "^2.0.0",
    "karma-cli": "^1.0.1",
    "karma-htmlfile-reporter": "^0.3.4",
    "karma-jasmine": "^1.0.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "lite-server": "^2.2.2",
    "lodash": "^4.16.4",
    "protractor": "4.0.9",
    "rimraf": "^2.5.4",
    "superagent": "^3.3.1",
    "tslint": "^3.15.1",
    "typescript": "^2.0.3",
    "webdriver-manager": "10.2.5"
  },
  "repository": {}
}

**Comment by pkozlowski-opensource** [comment]
See breaking changes section for alpha.18: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CHANGELOG.md#breaking-changes

tl;dr; you need to update your Bootstrap version if you want to use alpha.18.
